src/main/java/reposense/system/CommandRunner.java

```diff
@@ -5,9 +5,14 @@
 
 import reposense.util.SystemUtil;
 
+/**
+ * Contains command running related functionalities.
+ */
 public class CommandRunner {
 
-
+    /**
+     * Runs a command.
```

Comment:
`Spawns a backend terminal process, with working directory at {@code path}, to executes the {@code command}.`